pkg/storaged/filesystem/utils.jsx

```diff
@@ -85,6 +129,13 @@ export function is_valid_mount_point(client, block, val, format_only, for_fstab)
         return cockpit.format(_("Mount point is already used for $0"),
                               other_blocks.map(nice_block_name).join(", "));
 
+    if (subvol) {
```

Comment:
This should not be guarded `subvol`, ordinary filesystems can also try to use mount points of subvols by accident.

Reply:
But it should be guarded by `!format_only`, no?

Reply:
Good point, but is that not already handled `find_blocks_for_mount_point`? Seems this is mostly related to the comment above which needs fixing first.